src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/genericwhitespace/Example2.java

```diff
@@ -0,0 +1,36 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="GenericWhitespace"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.whitespace.genericwhitespace;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+// xdoc section -- start
+public class Example2 {
+  List< String> l; // violation, ''<' is followed by whitespace.'
```

Comment:
Let's have single example file, as config is without any properties.

So we will have:
```
// Generic type reference
    Map<String, List<Integer>> p;
    Map<String, List<Integer> > p; // violation 
```

So correct and bad code will be on adjacent lines, so clear  visual representation of what isnot good